Codeinwp/otter-blocks
# [Rock] Add more E2E tests

Closes #.


### Summary


Added E2E:

- Section
- Tabs
- Accordion
- Slider
- File loading for File e2e

----

### Test instructions


This is dev only. The code source was not touched, so nothing should happen with the build. Just see if it can be installed without error and a light check (like inserting a block in a new post). 



---- 

### Checklist before the final review
